lib/engine/game/g_1880/game.rb

```diff
@@ -684,9 +684,14 @@ def revenue_stops(route)
             unless stops.find { |stop| stop.tokened_by?(route.corporation) }
               stops.pop
               stops << sorted_stops.find { |stop| stop.tokened_by?(route.corporation) }
+              stops = stops.compact
             end
```

Comment:
L686 sometimes adds nil to stops when none of the stations are tokened.
This means that if a user starts the route away from the corp's tokened spots they get errors until the route has a token in it.
compact fixes it. (Is there a one liner that does << unless nil ?)